packages/gatsby/src/commands/types.ts

```diff
@@ -15,14 +16,15 @@ export interface IProgram {
   port: number
   proxyPort: number
   host: string
-  report: typeof reporter
+  report: Reporter
```

Comment:
somehow this reverts this PR:

- #24941 `refactor(gatsby): convert program reducer to typescript`

```diff
-  report: Reporter
+  report: typeof reporter
```

was this intended?

//cc @tgallacher